credmaster.py

```diff
@@ -560,7 +565,7 @@ def log_entry(entry):
 	fp_args.add_argument('--access_key', type=str, default=None, help='AWS Access Key')
 	fp_args.add_argument('--secret_access_key', type=str, default=None, help='AWS Secret Access Key')
 	fp_args.add_argument('--session_token', type=str, default=None, help='AWS Session Token')
-	fp_args.add_argument('--config', type=str, default=None, help='Authenticate to AWS using config file aws.config')
+	fp_args.add_argument('--config', type=str, default=None, help='Authenticate to AWS and add in webhooks too plus other config if wanted')
```

Comment:
Change to just say something like "config file for all configuration options" or something similar